packages/web/resolvers/utils/email.ts

```diff
@@ -255,11 +261,30 @@ const sendPremiumWelcomeEmail = ({ user }: sendPremiumWelcomeEmailArgs) => {
   })
 }
 
+const sendReportSpamPostEmail = ({
+  postId,
+  postAuthorId,
+  reportingUserId,
+}: sendReportSpamPostEmailArgs) => {
+  return sendJmail({
+    from: 'robin@journaly.com',
+    to: 'hello@journaly.com',
+    subject: `Spam Report: Post ${postId}`,
+    html: makeEmail(`
+      <h3>Spam Post Report:</h3>
+      <p>Post ID: ${postId}</p>
+      <p>Post Author ID: ${postAuthorId}</p>
+      <p>Reported by User ID: ${reportingUserId}</p>
```

Comment:
Linking to the post being reported, and the reporter and author, will make fielding these a bit less tedious. You can use `process.env.SITE_DOMAIN` to build an absolute URL for the link

Reply:
A great call! How do these look? For now I just used the legacy profile URLs since it's still fully supported and allows us to simply use the user IDs. Let me know if this looks good for now!